kolibri/plugins/learn/assets/src/views/LibraryPage/index.vue

```diff
@@ -296,6 +340,40 @@
         this.mobileSidePanelIsOpen = !this.mobileSidePanelIsOpen;
       },
     },
+    $trs: {
+      otherLibraries: {
+        message: 'Other libraries',
+        context: 'Header for viewing other remote content Library',
+      },
+      searchingOtherLibrary: {
+        message: 'Searching for libraries around you.',
+        context: 'Connection state for showing other library',
+      },
+      noOtherLibraries: {
+        message: 'No other libraries around you right now',
+        context: 'Connection state when there is no other libraries around',
+      },
+      showingAllLibraries: {
+        message: 'Showing all available libraries around you.',
+        context: 'Connection state when the device is connected and shows other libraries',
+      },
+      refresh: {
+        message: 'Refresh',
+        context: 'Link for refreshing library',
+      },
+      viewMore: {
+        message: 'View More',
```

Comment:
Note that we already have this string in `commonCoreStrings` so we don't need to add this here!